DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+%% LyX 2.2.3 created this file.  For more info, see http://www.lyx.org/.
+%% Do not edit unless you really know what you are doing.
+\documentclass[DM,lsstdraft,toc,authoryear]{lsstdoc}
+\ifx\hypersetup\undefined
+  \AtBeginDocument{%
+    \hypersetup{unicode=true}
+  }
+\else
+  \hypersetup{unicode=true}
+\fi
+
+\makeatletter
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LyX specific LaTeX commands.
+
+\title[jointcal]{jointcal: Simultaneous Astrometry \& Photometry for thousands of
+Exposures with Large CCD Mosaics}
+\author{John Parejko (University of Washington), Pierre Astier (LPNHE/IN2P3/CNRS
+Paris)}
+\setDocRef{DMTN-036}
+\date{\today}
+\setDocAbstract{The jointcal package simultaneously optimizes the astrometric and
+photometric calibrations of a set of astronomical images. In principle
+and often in practice, this approach produces distortion and thoroughput
+models which are more precise than when fitted independently. This
+is especially true when the images are deeper than the astrometric
+reference catalogs. In the ``Astromatic'' software suite, this simultaneous
+astrometry functionality is fulfilled by ``SCAMP''. The code we
+describe here has similar aims, but follows a slightly different route.
+Jointcal is built on top of the the LSST Data Management software
+stack.}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% User specified LaTeX commands.
+\newcommand{\ClName}[1]{{\bf #1}}
+\newcommand{\RoutineName}[1]{\texttt{#1}}
+%\def\arcsec{\hbox{$^{\prime\prime}$}}
+%\providecommand{\eprint}[1]{\href{http://arxiv.org/abs/#1}{#1}}
+
+% Change history defined here.
+% Order: oldest first.
+% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
+% See LPM-51 for version number policy.
+\setDocChangeRecord{%
+  \addtohist{1}{2017-09-01}{Unreleased.}{John Parejko}
+}
+
+\makeatother
+
+\usepackage{xunicode}
```

Comment:
This package is already loaded by the class.

Reply:
That's an auto-insert by LyX. I'll ask them about adding it to the list of things that can be defined as class-provided in the layout.

Reply:
Upstream request sent. I'll leave this as is for now (it doesn't break the build).

Reply:
Upstream response is that this will be a configurable in the layout in LyX 2.3.